### PR TITLE
Bugfix/creating entry

### LINKED ISSUE #45: error creating a entry
I tried to create a entry and this message appeared

Screen Shot 2019-12-15 at 22 09 04

### PR DESCRIPTION
**Pull request by renatamarques97**
resolves #45 

previous behavior 
Screen Shot 2019-12-15 at 22 09 04

new behavior
Screen Shot 2019-12-15 at 22 32 38

User must be submitted in form, so I added a hidden field with current user

### CODE REVIEW

**Comment by marclerodrigues** [comment]
Could you please take a look into why your build is broken?

File: spec/features/entries/form_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe 'form', type: :feature do
+  describe 'submit a form' do
```
Comment (marclerodrigues) [inline spec/features/entries/form_spec.rb]: Are you still working in this spec?

Comment (renatamarques97) [reply spec/features/entries/form_spec.rb]: yes

Comment (renatamarques97) [reply spec/features/entries/form_spec.rb]: don't worry, I'll take care of this

**Comment by marclerodrigues** [comment]
With the recent merges we had a little bit of conflict, could you please take a look into it? Overall this is looking REALLY good. Congrats!

File: app/views/entries/_form.html.erb [hunk 2]
```diff
@@ -12,25 +12,31 @@
...
 
   <div class="field">
-    <%= form.label t(:day, scope: 'activerecord.attributes.entries') %>
+    <%= form.label :day, t(:day, scope: 'activerecord.attributes.entries') %>
     <%= form.date_field :day %>
   </div>
 
+  <% if user_signed_in? %>
+    <div class="field">
+      <%= form.hidden_field :user_id, value: current_user.id %>
```
Comment (sobrinho) [inline app/views/entries/_form.html.erb]: I don't like this, a bad user could change the value of the hidden field to any desired user.

It is better to have something like this in the controller:

```ruby
def create
  @entry = Entry.new(entry_params)
  @entry.user = current_user
  @entry.save

  respond_with @entry
end
```

Comment (jeduardo824) [reply app/views/entries/_form.html.erb]: I agree

File: spec/features/entries/form_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    let(:user) { create(:user) }
+
+    before do
+      sign_in user
+      visit new_entry_path
+
+      select "Feliz", from: 'entry_feeling'
```
Comment (jeduardo824) [inline spec/features/entries/form_spec.rb]: Try to use the project standards about quotation marks.

File: app/controllers/entries_controller.rb [hunk 4]
```diff
@@ -17,7 +17,7 @@ def edit
   end
 
   def create
-    @entry = current_user.entries.new(entry_params)
+    @entry = current_user.entries.new(entry_params.merge(user_id: current_user.id ))
```
Comment (sobrinho) [inline app/controllers/entries_controller.rb]: extra space before the `))`.

**Comment by renatamarques97** [comment]
@marclerodrigues please take a look and merge if everything is okay, please :)

**Pull request merged**